site/docs/numbers/porting/port-ins/stand-alone-port-ins/tollFreePortabilityValidations.mdx

```diff
@@ -0,0 +1,324 @@
+---
+id: tollFreePortabilityValidations
+title: Toll Free Portability Validations
+sidebar_label: Toll Free Portability Validations
+slug: /numbers/porting/port-ins/stand-alone-portins/tollFreePortingValidations
+description: How to check if toll free numbers are portable
+keywords:
+  - porting
+  - portin
+  - port-in
+  - port
+  - lnp
+  - portability
+  - toll free
+  - checker
+  - check
+hide_title: false
+image: "@site/static/img/bw-icon.svg"
+---
+
+# Checking Portability for Toll Free Numbers
+
+In the past, Bandwidth did not provide a means to determine toll free number status and provider information. We are happy to say that that is no longer the case.
+
+Bandwidth provides the [/accounts/{accountId}/tollFreePortingValidations](/apis/numbers/#operation/CreateTollFreePortingValidation) endpoint to allow you to retrieve official toll free number status and provider information for up to 5000 toll free numbers per invocation. The input is simply a list of toll free numbers, and the output describes which numbers are portable, manually portable, and non-portable.
+
+This endpoint is available to help you submit toll free port-in requests to Bandwidth that are more likely to succeed. It will help you avoid submitting non-portable numbers, or numbers that must be ported in separate port requests because they are hosted by different providers.
+
+Because it can take several minutes to retrieve toll free number information from SOMOS (the North American toll free number administrator), the /tollFreePortingValidations API is asynchronous. This means that your request will create an order that you can either poll for results, or you may subscribe to notifications about state changes. Please refer to [Order Model](/docs/numbers/porting/orderModel.mdx) for details.
+
+Note: You may skip this request and proceed directly to submission of a port-in request, using POST /accounts/{accountId}/portins](/apis/numbers/#operation/CreatePortin). This will cause Bandwidth to perform the same validation. If the request contains non-portable toll free numbers, or numbers that cannot be ported together, the port-in will transition to an INVALID_TFNS state.
```

Comment:
```suggestion
Note: You may skip this request and proceed directly to submission of a port-in request, using [POST /accounts/{accountId}/portins](/apis/numbers/#operation/CreatePortin). This will cause Bandwidth to perform the same validation. If the request contains non-portable toll free numbers, or numbers that cannot be ported together, the port-in will transition to an INVALID_TFNS state.
```